### ISSUE TITLE
Java wildcard parameter types are unhandled in javacp

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#1653]
Javacp currently emits a dummy symbol `local_wildcard` for Java wildcard symbols. This limitation has been invisible so far but after #1650 the diff expect tests grew with the following entry caused by this bug

```diff
+=====================================
+com.javacp.Test#wildcard().(wildcard)
+=====================================
+--- metac
++++ metacp
+ signature {
+   valueSignature {
+     tpe {
+-      existentialType {
+-        tpe {
++      typeRef {
++        prefix {
++        symbol: "java.util.ArrayList#"
++        type_arguments {
+           typeRef {
+             prefix {
+-            symbol: "java.util.ArrayList#"
+-            type_arguments {
+-              typeRef {
+-                prefix {
+-                symbol: "local0"
++            symbol: "local_wildcard"
+-        declarations {
+-          hardlinks {
+-            symbol: "local0"
+-            kind: TYPE
+-            properties: 4
+-            name: "_"
+-            accessibility {
+-              tag: PUBLIC
+-              symbol: ""
+-            language: SCALA
+-            signature {
+-              typeSignature {
+-                type_parameters {
+-                lower_bound {
+-                  typeRef {
+-                    prefix {
+-                    symbol: "scala.Nothing#"
+-                upper_bound {
+-                  typeRef {
+-                    prefix {
+-                    symbol: "scala.Any#"
```

Question is, should we update the spec to have a first-class representation of Java wildcard types or should we desugar wildcards into existential types like Scalac does?

**Comment by xeno-by** [comment]
Superseded by https://github.com/scalameta/scalameta/issues/1703.
